src/tasks/package.js

```diff
@@ -29,32 +29,33 @@ const packages = [
 module.exports = (config) => {
   json('package.json')
     .merge({
+      main: 'dist/cjs.js',
+      files: [
+        'dist',
+      ],
       engines: {
         // Some versions are skipped because of known issues, see https://github.com/webpack-contrib/organization/issues/7
         node: `>= ${config.minNode} < 5.0.0 || >= 5.10`,
       },
       scripts: {
-        start: 'yarn run serve:dev src',
-        build: "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
-        'clean:dist': 'del-cli dist',
-        lint: 'eslint --cache src test',
+        start: 'yarn run build -- -w',
+        build: 'cross-env NODE_ENV=production babel src -d dist',
```

Comment:
We should decide upon either of, in `src` is also fine but it's just a structural thing, which can be consistent. Personally a separate test folder is cleaner imho.